docs/buckets.rst

```diff
@@ -0,0 +1,99 @@
+Buckets
+#######
+
+Buckets are a group of collections, shared between users, with fined
+permissions on the data stored inside.
+
+Basically a bucket have got an id and a list of people identifier that
+can administrate it.
```

Comment:
I still believe we shouldn't go with this solution. We should have permissions associated.